### PR TITLE
@dzucconi => Artist exhibition highlights

### PR DESCRIPTION
**Pull request by mzikherman**
Something like:

```
{
  "data": {
    "artist": {
      "id": "pablo-picasso",
      "name": "Pablo Picasso",
      "exhibition_highlights": [
        {
          "id": "centre-for-fine-arts-bozar-picasso-sculptures"
        },
        {
          "id": "musee-picasso-paris-picasso-an-anniversary-exhibition"
        },
        {
          "id": "musee-picasso-paris-picasso-sculptor"
        },
        {
          "id": "museum-of-modern-art-picasso-sculpture"
        },
        {
          "id": "the-metropolitan-museum-of-art-unfinished-thoughts-left-visible"
        }
      ]
    }
  }
}
```

### CODE REVIEW

File: schema/artist/index.js [hunk 1]
```diff
@@ -170,6 +170,93 @@ const ArtistType = new GraphQLObjectType({
 
       statuses: ArtistStatuses,
 
+      exhibition_highlights: {
```
Comment (mzikherman) [inline schema/artist/index.js]: 9 API calls :neutral_face:

Comment (dzucconi) [inline schema/artist/index.js]: LOL

File: schema/artist/index.js [hunk 2]
```diff
@@ -170,6 +170,93 @@ const ArtistType = new GraphQLObjectType({
...
+              solo_show: false,
+              at_a_fair: false
+            }),
+            gravity('related/shows', {
+              artist_id: id,
+              sort: '-end_at',
+              displayable: true,
+              solo_show: true,
+              at_a_fair: true
+            }),
```
Comment (dzucconi) [inline schema/artist/index.js]: Wait seriously??? What's the spec for this

Comment (dzucconi) [inline schema/artist/index.js]: Oh I see your comment

File: schema/artist/index.js [hunk 3]
```diff
@@ -170,6 +170,93 @@ const ArtistType = new GraphQLObjectType({
...
       statuses: ArtistStatuses,
 
+      exhibition_highlights: {
+        type: new GraphQLList(PartnerShow.type),
+        resolve: ({ id }, options) => {
+          // First, highest tier solo institutional shows, and then group.
+          // Second, highest tier solo gallery shows, and then group.
+          // Third, lower tier solo institutional shows, and then group.
+          // Fourth, lower tier solo gallery shows, and then group.
+          // Last fair booths.
```
Comment (dzucconi) [inline schema/artist/index.js]: Maybe inline these comments with the reqs for clarity?

File: schema/artist/index.js [hunk 4]
```diff
@@ -170,6 +170,93 @@ const ArtistType = new GraphQLObjectType({
...
+              artist_id: id,
+              sort: '-end_at',
+              displayable: true,
+              solo_show: true,
+              at_a_fair: true,
+            }),
+          ]).then(([shows, shows2, shows3, shows4, shows5, shows6, shows7, shows8, shows9]) => {
+            const allShows = concat(shows, shows2, shows3, shows4, shows5, shows6, shows7, shows8, shows9);
+            return take(allShows, 5);
+          });
```
Comment (dzucconi) [inline schema/artist/index.js]:
```javascript
.then(allShows => take(concat(...allShows), 5)
```

Comment (dzucconi) [inline schema/artist/index.js]: :golf:

**Comment by dzucconi** [comment]
Allllright

**Pull request merged**